main.go

```diff
@@ -125,6 +126,11 @@ func wrapMain() error {
 	vargs := GAE{}
 	workspace := ""
 
+	// If there is an env vars file load it
+	if _, err := os.Stat("/run/drone/env"); err == nil {
+		godotenv.Load("/run/drone/env")
```

Comment:
I don't think this is will do what you are expecting to do: it will try to overload the environment from that file, but your gae credentials stored in the current workspace are not loaded.